apps/arm/src/main.cpp

```diff
@@ -167,14 +170,72 @@ static mbed_error_status_t setToolTipDeployment(CANMsg &msg) {
   }
 }
 
-// Enable or disable PID tuning mode10
+// Enable or disable PID tuning mode
+// Manually send CAN message to SET_PID_TUNING_MODE with appropriate boolean
 static mbed_error_status_t setPIDTuningMode(CANMsg &msg) {
-  return MBED_SUCCESS;  // TODO
+  msg.getPayload(allowPIDParamTuning);
+  return MBED_SUCCESS;
 }
 
 // Configure PID parameters
 static mbed_error_status_t setPIDParameter(CANMsg &msg) {
```

Comment:
Need to be very careful here. can thread and main thread both accessing actuator controller class at the same time and I don't see any locks etc. Look at actuator controller code. I'm pretty sure its not inherently thread-safe.

Reply:
ive addressed this in the latest version